docs/eventing/channels/subscriptions.md

```diff
@@ -23,26 +30,38 @@ kn subscription create <subscription_name> \
 ```
 
 
-- `--channel` specifies the source for cloud events that should be processed. You must provide the channel name. If you are not using the default channel that is backed by the Channel resource, you must prefix the channel name with the `<Group:Version:Kind>` for the specified channel type. For example, this will be `messaging.knative.dev:v1beta1:KafkaChannel` for a Kafka backed channel.
+- `--channel` specifies the source for cloud events that should be processed.
```

Comment:
CloudEvents

I just wonder if we should use either 'event' or 'CloudEvent' in all the places we talk about theM/

Reply:
I think we can just use "events" all over as you mentioned. And leave the "CloudEvents" to when we want to be specific about the format.

Reply:
Let's not block the PR on this and instead add it to the [word list](https://docs.google.com/spreadsheets/d/1p1_kBUd6ZvonxHkMcEJPayf6QIpExuFf5cFq0ptar7I/edit#gid=0)?